apps/www/pages/pricing/CHANGELOG.txt

```diff
@@ -1,5 +1,10 @@
 Pricing Changelog
 
+### 24 October 2022
```

Comment:
@MildTomato I saw you did this for Edge Functions. Let me know if this is still the practice.